Makefile

```diff
@@ -279,6 +279,8 @@ bundle: manifests operator-sdk kustomize ## Generate bundle manifests and metada
 	sed -e 's/annotations://' -e 's/  /LABEL /g' -e 's/: /=/g'  config/metadata/$(BUNDLE_PACKAGE).annotations.yaml >> bundle.Dockerfile
 	sed -i.bak 's/operators.operatorframework.io.bundle.package.v1:.*/operators.operatorframework.io.bundle.package.v1: $(BUNDLE_ANNOTATION_PACKAGE)/' bundle/metadata/annotations.yaml && rm bundle/metadata/annotations.yaml.bak
 	sed -i.bak 's/operators.operatorframework.io.bundle.package.v1=.*/operators.operatorframework.io.bundle.package.v1=$(BUNDLE_ANNOTATION_PACKAGE)/' bundle.Dockerfile && rm bundle.Dockerfile.bak
+	rm bundle/manifests/activemq-artemis-webhook-service_v1_service.yaml
```

Comment:
Why do you need to remove a file?

Reply:
This service is not needed when olm can do it for the operator.